python/ray/tests/test_tempfile.py

```diff
@@ -3,14 +3,14 @@
 from __future__ import print_function
 
 import os
-import shutil
 import time
 import pytest
 import ray
 from ray.tests.cluster_utils import Cluster
+from ray.tests.utils import try_remove_directory
 
 
-def test_conn_cluster():
+def test_conn_cluster(shutdown_only):
```

Comment:
I tracked down the source of the error, and this is not it. I think we should remove this fixture.

Reply:
Ray cannot be initialized in this test, so I think we should not do `shutdown_only` to also test that it is not being initialized.